packages/ui-library/src/components/navigation/index.tsx

```diff
@@ -335,7 +335,12 @@ const Navigation = ({
             </IconButton>
             <AppBarRightContainer alignItems="center" justifyContent="flex-end" flex={1} gap={2}>
               {extraHeaderTools}
-              <OptionsMenu options={helpOptions} mainDisplay={<HelpIcon />} />
+              <OptionsMenu
+                options={helpOptions}
+                customClassname="beamer-whats-new"
```

Comment:
Is it necessary to have this class here? It's already applied to the option item

Reply:
We need this so the badge also shows up on the help icon